mu/logic.py

```diff
@@ -410,7 +410,7 @@ def check_pycodestyle(code):
     os.close(code_fd)
     save_and_encode(code, code_filename)
     # Configure which PEP8 rules to ignore.
-    ignore = ('E121', 'E123', 'E126', 'E226', 'E302', 'E305', 'E24', 'E704',
+    ignore = ('E121', 'E123', 'E126', 'E226', 'E203', 'E302', 'E305', 'E24', 'E704',
```

Comment:
This is necessary since Black formats slices according to PEP 8, and not according to what pycodestyle prefers:

![screenshot 2018-10-17 12 02 29](https://user-images.githubusercontent.com/55281/47380704-f3c67000-d6f5-11e8-8e92-fe84b63cdf96.png)